src/window.rs

```diff
@@ -380,6 +395,40 @@ impl Window {
     pub fn set_decorations(&self, decorations: bool) {
         self.window.set_decorations(decorations)
     }
+    
+    /// Centers the window on the current monitor the window lives in.
```

Comment:
Please specify the platform-dependent behavior of this function. For example it cannot work at all on Wayland, as the app cannot control its placement on the monitor.